### ISSUE TITLE
Platystencephaly

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5682]
**Preferred term label:**

Platystencephaly
**Synonyms**

**Definition (free text, please give PubMed ID)**
Extreme width of the skull in the occipital region, with anterior narrowing and prognathism.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/31839203/
(1/3)

**Your nano-attribution (ORCID)**
